chunjun-connectors/chunjun-connector-jdbc-base/src/main/java/com/dtstack/chunjun/connector/jdbc/source/JdbcInputFormat.java

```diff
@@ -349,21 +294,27 @@ protected void initMetric(InputSplit inputSplit) {
      *
      * @param inputSplit 数据分片
      */
-    protected void getMaxValue(InputSplit inputSplit) {
+    protected String getMaxValue(InputSplit inputSplit) {
         String maxValue;
         if (inputSplit.getSplitNumber() == 0) {
-            maxValue = getMaxValueFromDb();
+            maxValue = incrementKeyUtil.transToLocationValue(getMaxValueFromDb()).toString();
```

Comment:
Here, I have a concern about whether the returned maxval value should be+1 when pollingFromMax=true. Otherwise, the last item in the history will be consumed when consumption starts (it should not be consumed)

这里就是，我有一个担心，就是 当pollingFromMax=true 时候，这个返回的 maxval值是否要+1 ，不然，开始消费的时候 会把历史记录的最后一条也消费的（应该是不消费的 ）
我今天自己的实现 是 用了一个 这样的方法 https://github.com/qlangtech/chunjun/blob/61a1f09bd31bcb3c7528a55178a835db6d4e81ff/chunjun-connectors/chunjun-connector-jdbc-base/src/main/java/com/dtstack/chunjun/connector/jdbc/source/JdbcInputFormat.java#L196,

我之前测试的时候 是用 “where #column >= #val” 的，改过了？其实会将历史最后一条也消费的（其实并不需要）

Reply:
> Could you please tell me the reason. In my opinion, the param 'pollingFromMax' and 'useMaxFunc' is not conflict with each other.
Can you list how you expect the program to behave when 'pollingFromMax' and 'useMaxFunc' are set to various values (there should be four combinations here)?

Reply:
We can imagine that if polling keeps consuming the last piece of data (>=), does it mean that the last piece of data may be consumed multiple times
Why don't we just use '>' instead of '+1' if we don't want to consume the last piece of data over and over again (' +1 'doesn't work for double data in my opinion)